src/personality_test1.c

```diff
@@ -396,13 +397,14 @@ static void PersonalityTest_DisplayStarterSprite(void)
     CallPrepareTextbox_8008C54(1);
     sub_80073B8(1);
 
-    faceFile = GetDialogueSpriteDataPtr(starterID);
-    r6 = ((struct FaceData *)faceFile->data)->unk0[1 + EMOTION_HAPPY];
-    faceIndex = EMOTION_HAPPY;
-    for (palleteIndex = 0; palleteIndex < 0x10; palleteIndex++)
-        SetBGPaletteBufferColorArray(palleteIndex + 0xE0, &((struct FaceData *)faceFile->data)->unk0[faceIndex][palleteIndex * 4]);
+    faceFile = (void*) GetDialogueSpriteDataPtr(starterID);
+    gfx = faceFile->data->sprites[1].gfx;
+    emotionId = 1;
```

Comment:
You lost the macro here

Reply:
The emotion one? Yeah, I forgot to update it.